src/js/components/Box/__tests__/Box-layout-test.tsx

```diff
@@ -229,6 +229,23 @@ describe('Box', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('row and column gap', () => {
+    const { container } = render(
+      <Grommet>
+        <Box gap={{ row: 'xlarge', column: '30px' }} direction="row">
+          <Box />
+          <Box />
+        </Box>
+        <Box as="span" gap="small">
+          <span>first</span>
+          <span>second</span>
+        </Box>
+      </Grommet>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
```suggestion
    expect(asFragment()).toMatchSnapshot();
```
more recently, we've been using `asFragment` instead of container -- can you regenerate snapshots when you merge this?